desktop/apps/consignments2/client/actions.js

```diff
@@ -454,7 +464,6 @@ export function uploadImageToGemini (key, bucket, fileName) {
       dispatch(uploadImageToConvection(token, fileName))
     } catch (err) {
       dispatch(updateError('Unable to process image.'))
-      dispatch(stopProcessingImage(fileName))
```

Comment:
Since the "processing" state is more of a "pending" state, we should leave images in their `processing` state even if an error occurs (per the GIF above).